### PR TITLE
[updated] Make event alliances endpoint cronjob run more often

### PR DESCRIPTION
**Pull request by phil-lopreiato**
Takes #1239 and implemented suggestion from review.

Closes #1239 also

### CODE REVIEW

File: controllers/datafeed_controller.py [hunk 1]
```diff
@@ -105,9 +105,13 @@ class FMSAPIEventAlliancesEnqueue(webapp.RequestHandler):
     Handles enqueing getting alliances
     """
     def get(self, when):
+        events = []
```
Comment (gregmarra) [inline controllers/datafeed_controller.py]: Why is this needed? `events` will always ben set in the if below.

Comment (phil-lopreiato) [inline controllers/datafeed_controller.py]: This does not appear to be needed, want me to remove it? (I didn't write the original PR)

Comment (fangeugene) [inline controllers/datafeed_controller.py]: Yeah remove it.

**Comment by phil-lopreiato** [comment]
All set
